objectives.md

```diff
@@ -0,0 +1,35 @@
+# Part 1: The Basics 
+
+| **Objective**                                                                                                         | **Rationale**                                                                                                                                                       |
+|-----------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Interpret the same dataset using raw numbers and data visualizations within a time limit                              | Data visualizations leverage our brain’s ability to quickly recognize visual relationships, making it easier to recognize patterns, trends, and anomalies in the data. |
+| Code a simple scatterplot by encoding data variables as visual aesthetics and representing values as geometric marks. | The Grammar of Graphics approach provides a powerful set of rules and components for building data visualizations, used by coding frameworks such as Altair and ggplot. |
+| Apply the concept of layering to recreate a complex visualization                                                      | Layering different geometric marks on the same canvas allows you to construct a wider variety of data visualizations.                                                   |
```

Comment:
@rorywhite200  Here are my thoughts on what a few LOs and exercises for this chapter could look like. They include your first two so let's start with this when you draft the first chapter

1. Why do we need visualizations? What are the advantages compared to showing data as numerical summaries?
    - Charts are a superior means of communicating data to humans. It is advantageous to text/numbers due to our evolved pattern recognition ability which allows us to quicker get the "gist"/understanding of the data by looking at a visual representation of it. The choices we make when creating visualizations also allows us to highlight different messages in the data, which would not stand out when looking at the raw data and often not when looking at numerical summaries either, since many different data sets can have the same numerical summaries.
    - LO: (explain) the main advantages of  visualizing data instead of presenting it with numbers is that they are easier to interpret for humans
2. What is a grammar of graphics?
    - A grammar of graphics defines the grammatical rules that can be used to compose entire visualization from smaller building blocks. refers to that there are small/atomic semantic components that can be combined together to create visualizations. This is akin to a natural language grammar where words can be composed into sentences according to a predefined set of rules. Depending on how the words/graphical elements are put together, the sentence/visualization will rely a different message.
    - LO: (describe)  A grammar of graphics defines the grammatical rules that can be used to compose entire visualization from smaller building blocks.
3. How can you create a scatter plot using the grammar of graphics in Altair/ggplot
    - `alt.Chart(df).mark_point().encode(x='a', y='b')`
    - `ggplot(df) + aes(x=a, y=b) + geom_point()`
    - LO: (apply) Be able to use the visualization grammars Altair and ggplot and create a basic chart via `alt.Chart()...` / `ggplot()...`
4. What is the advantage of using a high level / declarative syntax instead of  a low level / imperative for creating visualizations?
    - A high level level syntax allows us to think in terms of the data, and have the visualization library takes care of things such as choosing appropriate colors and assigning them two groups without us having to specifically loop through the df.
        - LO: (explain) The advantage of high level level syntax allows us to think in terms of the data, rather than graphical details.

---

- Do not include in ch 1 when moving content from 531
    - Grouping by color size etc (~ch2)
    - Aggregation average (~ch3)
    - Line charts (~ch3)
    - Unsure: layering + concat?
        -  This would be nice in terms of understanding the concept of a visualization **grammar**. I'm just unsure if we will be able to demonstrate it effectively before introducing a different mark such as line so that learners can see line + point in the chart. Maybe? ggplot would also require cowplot for concatenation, so a bit non-ideal...